Wireframe/index.html

```diff
@@ -4,9 +4,66 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="/Wireframe/css/style.css">
   </head>
   <body>
+    <main> 
+      <article class="git-defenation"> 
+      <h2>what is Git?</h2>
+      <p>Git is a tool that helps people work together on computer programs and files.
+         It's like a smart way to keep track of changes in your work and collaborate with others.</p>
+  
+         <p> 
+        Imagine you're working on a group project where everyone needs to add their own ideas and changes to a document. 
+        Without Git, it might get messy, with people emailing different versions back and forth. But with Git, it's much easier.
+        Here's how it works: Git keeps a record of all the changes you make to your project.</p>
+        <p> 
+         It's like a time machine that lets you go back to any previous version of your work if something goes wrong or if you want to see how it looked before.
+          This means you can experiment and make changes without worrying about messing things up.
+        Plus, Git makes it simple for multiple people to work on the same project at the same time.
+         It keeps track of who made what changes and helps merge them together smoothly. </p>
+        
+        <button class="read-button">Read more</button>
+      </article>
+      
+    <section id="git-main"> 
+      <article id="article1" class="article2" >
```

Comment:
Can you talk through your choices here with class names? Is there a clearer name you could choose, that would explain what article2 means? (I see it can't mean second article as the third article also has class article2)